ci-operator/step-registry/rosa/sts/cluster/machinepool/rosa-sts-cluster-machinepool-commands.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+# Get cluster 
+CLUSTER_ID=$(cat "${SHARED_DIR}/cluster-id")
+echo "CLUSTER_ID is $CLUSTER_ID"
+
+# Configure aws
+CLOUD_PROVIDER_REGION=${LEASED_RESOURCE}
+if [[ "$HOSTED_CP" == "true" ]] && [[ ! -z "$REGION" ]]; then
+  CLOUD_PROVIDER_REGION="${REGION}"
+fi
+
+AWSCRED="${CLUSTER_PROFILE_DIR}/.awscred"
+if [[ -f "${AWSCRED}" ]]; then
+  export AWS_SHARED_CREDENTIALS_FILE="${AWSCRED}"
+  export AWS_DEFAULT_REGION="${CLOUD_PROVIDER_REGION}"
+else
+  echo "Did not find compatible cloud provider cluster_profile"
+  exit 1
+fi
+
+# Log in
+ROSA_VERSION=$(rosa version)
+ROSA_TOKEN=$(cat "${CLUSTER_PROFILE_DIR}/ocm-token")
+if [[ ! -z "${ROSA_TOKEN}" ]]; then
+  echo "Logging into ${ROSA_LOGIN_ENV} with offline token using rosa cli ${ROSA_VERSION}"
+  rosa login --env "${ROSA_LOGIN_ENV}" --token "${ROSA_TOKEN}"
+  if [ $? -ne 0 ]; then
+    echo "Login failed"
+    exit 1
+  fi
+else
+  echo "Cannot login! You need to specify the offline token ROSA_TOKEN!"
+  exit 1
+fi
+
+# Check if this is a HCP cluster
+is_hcp_cluster="$(rosa describe cluster -c "$CLUSTER_ID" -o json  | jq -r ".hypershift.enabled")"
+log "hypershift.enabled is set to $is_hcp_cluster"
+
+function rosaCreateMachinepool() {
```

Comment:
Do we still need this now that we are doing it only for HCP?

Reply:
no, not required

Reply:
Ok, so deleting I assume?